### DISCUSSION TITLE
Which plugin shows class/function name as the first row?

### DISCUSSION THREAD

**Discussion by karambaq** [crivotz/nv-ide#39] [Q&A] [answered]
<img width="637" alt="image" src="https://github.com/crivotz/nv-ide/assets/14233969/43b99614-c361-4bcf-ba94-11dd41919190">

I mean actual function signature is on the first line, but it still showing when I'm scrolling down, is it navic plugin?

**Comment by crivotz** [comment, marked as answer]
It is [nvim-treesitter-context](https://github.com/nvim-treesitter/nvim-treesitter-context)
Navic is used only in winbar for code breadcrumbs
